sdv/data_processing/data_processor.py

```diff
@@ -450,9 +455,17 @@ def _create_config(self, data, columns_created_by_constraints):
         sdtypes = {}
         transformers = {}
 
+        columns_in_multi_col_transformer = {
+            column for key in self.columns_to_mutli_col_transformer for column in key
+        }
```

Comment:
I wouldn't encourage this kind of list comprehension as it makes it a bit complicated to read, there is `chain` to help you achieve the same:
```python
from itertools import chain
columns_in_multi_col_transformer = list(self.columns_to_mutli_col_transformer)
columns_in_multi_col_transformer = set(chain(*columns_in_multi_col_transformer))
```

----
Here is an example
```python

In [7]: x = {
   ...:     ('a', 'b', 'c'): 'Transformer',
   ...:     ('d', 'b', 'k'): 'AnotherTransformer'
   ...: }

In [8]: from itertools import chain

In [9]: d = list(x)

In [10]: set(chain(*d))
Out[10]: {'a', 'b', 'c', 'd', 'k'}
```